src/db/add_package.rs

```diff
@@ -443,6 +445,8 @@ fn get_owners(pkg: &MetadataPackage) -> Result<Vec<CrateOwner>> {
     if let Some(owners) = json.as_object()
         .and_then(|j| j.get("users"))
         .and_then(|j| j.as_array()) {
+            result.reserve(owners.len());
```

Comment:
This indentation doesn't match the for loop below.

Reply:
Well, the indentation of that whole block is off, so whatever's fine